.github/workflows/format-prettier.yml

```diff
@@ -0,0 +1,52 @@
+name: Format with Prettier
+
+on:
+  push:
+    branches: [ 'main' ]
```

Comment:
We may want to trigger the workflow when pushing to `dev` branch.

Reply:
I am concerned that this might result in significant changes to the current dev branch, which could potentially cause confusion for team members who have not yet updated. Perhaps it would be prudent to confirm with everyone first.